src/main/scala/io/iohk/ethereum/blockchain/sync/regular/BlockFetcher.scala

```diff
@@ -40,7 +40,7 @@ class BlockFetcher(
     val syncConfig: SyncConfig,
     val blockValidator: BlockValidator
 ) extends Actor
-    with ActorLogging {
+    with Logger {
```

Comment:
`ActorLogging` should stay for all actors

Reply:
will revert the commit